src/commands/ListProtectedRoomsCommand.ts

```diff
@@ -24,7 +24,7 @@ export async function execListProtectedRooms(roomId: string, event: any, mjolnir
     let text = `Protected rooms (${rooms.length}):\n`;
 
     let hasRooms = false;
-    for (const protectedRoomId in rooms) {
+    for (const protectedRoomId of rooms) {
```

Comment:
Ouch, good catch!
How did we let that pass?

Reply:
Yep, this was unfortunate.